docs/versions.yaml

```diff
@@ -8,5 +8,9 @@
     - "en"
 "v0.0.4":
```

Comment:
Should we leave this one or remove it and remove the tag?

Reply:
i removed it